content/blog/managing-google-workspace-with-terraform.md

```
@@ -0,0 +1,297 @@
+callout: <p>👋 <b>If you're ready to take your infrastructure to the next level, we're here to help. We love to work together with engineering teams to help them build well-documented, scalable, automated IaC that make their jobs easier. <a href='/contact'>Get in touch!</a>
+---
+
+## Table of Contents
+
+- [Growing Pains](#growing-pains)
+- [Why This Matters for Scaling Teams](#why-this-matters-for-scaling-teams)
+- [Managing Google Workspace with Terraform](#managing-google-workspace-with-terraform)
+- [Getting Started With the Module](#getting-started-with-the-module)
+- [Module Design Decisions](#module-design-decisions)
+  - [Implementing Integration and Validation Tests](#design-decision-1---implementing-integration-and-validation-tests)
+  - [Choosing Intuitive Terraform Variable Structure](#design-decision-2---choosing-intuitive-terraform-variable-structure)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)